cudax/include/cuda/experimental/__memory_resource/shared_resource.cuh

```diff
@@ -51,15 +51,6 @@ struct shared_resource
 {
   static_assert(_CUDA_VMR::resource<_Resource>, "");
 
-  //! @brief Constructs a \c shared_resource referring to an object of type \c _Resource
-  //! that has been constructed with arguments \c __args. The \c _Resource object is
-  //! dynamically allocated with \c new.
-  //! @param __args The arguments to be passed to the \c _Resource constructor.
-  template <class... _Args>
-  explicit shared_resource(_Args&&... __args)
-      : __control_block(new _Control_block{_Resource{_CUDA_VSTD::forward<_Args>(__args)...}, 1})
-  {}
```

Comment:
i have a weak preference for keeping this constructor but making its first argument `cuda::std::in_place_type_t<_Resource>`.